portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/DataSet.path

```diff
@@ -230,6 +230,26 @@
 	<td>//div[label[contains(.,'Source Options')]]/select</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRESELECT_VALUE</td>
+	<td>//button[@class="dropdown-item"]//strong</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SELECT_FILTER_DATE</td>
+	<td>//div[contains(@class,"dropdown-menu date-picker-dropdown-menu show")]//select[contains(@class,'form-control form-control-sm') and contains(@name,'${key_type}')]</td>
+	<td></td>
```

Comment:
Hey @CarlosBrichete in this case I will have to stay with this path, because I couldn't find another one that would do the same action... but this way I can proceed with the test steps... ok ?